docs/transactions/iti41.md

```diff
@@ -1,35 +1,257 @@
+| DIS           | `440545006` _Prescription record_       | `urn:che:epr:ch-emed:dis:2022`             | `294121000195110` _Medication dispense document (record artifact)_ |
 | PADV          | `440545006` _Prescription record_       | `urn:che:epr:ch-emed:padv:2022`            | `419891008` _Record artifact_                                 |
 | PML           | `422735006` _Summary clinical document_ | `urn:che:epr:ch-emed:pml:2022`             | `721912009` _Medication summary document_                     |
-| PMLC          | `422735006` _Summary clinical document_ | `urn:che:epr:ch-emed:medication-card:2022` | ~~`721912009` _Medication summary document_~~<br>`736378000` _Medication management plan (record artifact)_ |
+| PMLC          | `422735006` _Summary clinical document_ | `urn:che:epr:ch-emed:medication-card:2022` |`736378000` _Medication management plan (record artifact)_ |
+
+The class and type codes are from the SNOMED CT system: `2.16.840.1.113883.6.96`. The system for the [`formatCode`](http://www.fhir.org/guides/stats/codesystem-ch.fhir.ig.ch-epr-term-2.16.756.5.30.1.127.3.10.10.html) is `2.16.756.5.30.1.127.3.10.10`
 
-The class and type codes are from the SNOMED CT system: `2.16.840.1.113883.6.96`.
+## Replacing a CH-EMED-EPR document
+
+The following rules must be observed when replacing a document :
+* [Patients](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-PAT) and their [representatives](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-REP) can only replace a document published by the same patient.
+	* representatives are not supported by the service yet.
+* [Healthcare professionals](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-HCP) (HCP) can only replace a document published by himself or by another HCP of same community of affiliation.
+    * Current implementation: an HCP can replace a document published by any other HCP. Further rules and possible implementations are under discussion.
+* [Document administrator](https://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html) may only replace a document published by an HCP of the same community of affiliation.
+	* Not implemented in PMP, to be discussed, for now a document admin can replace any document.
+* Replacing and replaced documents have to be of the same type (e.g. both MTP documents).
+* Replacing and replaced documents have to refer to the same patient.
+* Only documents at the end of a treatment chain can be replaced.
+* Replaced document must exist in the repository.
+* A replacement document must can be linked only to elements of the same medication chain. In case a PRE document is replaced, it may be linked only to elements of the medication chains of the referenced MTP. 
```

Comment:
```suggestion
* A replacement document must can be linked only to elements of the same medication chain. In case a PRE document is replaced, it may be linked only to elements of the medication chain of the referenced MTP. 
```